stable/minio/templates/post-install-create-bucket-pod.yaml

```diff
@@ -1,41 +1,46 @@
 {{- if .Values.defaultBucket.enabled }}
-apiVersion: v1
-kind: Pod
+apiVersion: batch/v1
+kind: Job
 metadata:
-  name: {{ template "minio.fullname" . }}-create-bucket
+  name: {{ template "minio.fullname" . }}-make-bucket-pod
   labels:
     app: {{ template "minio.fullname" . }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install,post-update
```

Comment:
It's not `post-update`. It should be `post-upgrade`. The error in the hook was not causing it to fire for me.

Reply:
great catch @mattfarina , thanks! I have updated the PR